WordPress/Tests/Functions/FunctionRestrictionsUnitTest.inc

```diff
@@ -0,0 +1,5 @@
+<?php
+
+wp_register_script( 'some-handle', 'http://example.com' ); // this is ok
```

Comment:
Please use a realistic example. As CDNs/loading of scripts from outside sources is forbidden/discouraged, this example is misleading.